AppController/lib/cron_helper.rb

```diff
@@ -129,17 +129,30 @@ def self.clear_app_crontab(app)
   private
 
 
-  # Adds a single line to this user's crontab. It is assumed that the line is
-  # correctly formatted in cron format.
+  # Checks if a crontab line is valid.
   #
   # Args:
-  #   line: A String that contains the line to add to root's crontab.
-  def self.add_line_to_crontab(line)
-    `rm crontab.tmp`
-    `crontab -l >> crontab.tmp`
-    `echo "#{line}" >> crontab.tmp`
-    `crontab crontab.tmp`
+  #   line: A String that contains a crontab line.
+  # Returns:
+  #   A boolean that expresses the validity of the line.
+  def self.valid_crontab_line(line)
+    crontab_exists = system('crontab -l')
+    if crontab_exists
+      `crontab -l > crontab.backup`
```

Comment:
Can this be run in parallel? If so, we may need to make the name unique (ie contab.$timestamp) ...